sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/AnalyzeRequestConverter.java

```diff
@@ -0,0 +1,94 @@
+package com.azure.search.documents.implementation.converters;
```

Comment:
Are these implementation files, auto-generated?
No license statements?

Reply:
Good catch! These are manually classes. It is supposed to have license. I will add license.